### PR TITLE
Pooja

### PR DESCRIPTION
**Pull request by PoojaGandhi123**

### CODE REVIEW

File: client/views/admin_templates.html [hunk 1]
```diff
@@ -60,7 +60,7 @@
 		{{$.Session.get 'adminError'}}
 	</div>
 	{{/if}}
-</template>
+</template> -->
```
Comment (desaiuditd) [inline client/views/admin_templates.html]: @PoojaGandhi123 Why did you comment this template out ? We need this

File: client/views/users/editView.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
               <div class="form-group">
                 <label class="col-md-4 control-label"></label>
                 <div class="col-md-8">
-                  <input type="submit" class="btn btn-large btn-success" value="Save HMIS Profile" />
+                  <input type="submit" class="btn btn-large btn-success" value="Sign in to Home" />
                 </div>
```
Comment (desaiuditd) [inline client/views/users/editView.html]: @PoojaGandhi123 This is not a login form. We are saving user details in HMIS profile here.

**Comment by PoojaGandhi123** [comment]
Okay. I will keep this in mind . Sorry! Will not happen again

File: client/helpers/client/searchClient.js [hunk 3]
```diff
@@ -62,6 +62,7 @@ Template.searchClient.helpers(
           const query = {};
           if (dataObject.isHMISClient) {
             query.query = `isHMISClient=true&schema=${dataObject.schema}`;
+            Router.go('preliminarySurvey', { _id: dataObject._id }, query);
```
Comment (desaiuditd) [inline client/helpers/client/searchClient.js]: @PoojaGandhi123 Why this line is added ?

We don't have any route names `preliminarySurvey`

**Comment by PoojaGandhi123** [comment]
Hey Udit,

I have removed this file but its on my local. The changes which I have to
push has all these in it


[image: --]

Pooja Gandhi
[image: https://]about.me/pooja.gandhi
<https://about.me/pooja.gandhi?promo=email_sig&utm_source=email_sig&utm_medium=external_link&utm_campaign=chrome_ext>

File: client/views/clients/clientForm.html [hunk 4]
```diff
@@ -4,35 +4,35 @@
 	<div class="form-group">
 		<label class="col-lg-3 control-label">First name:</label>
 		<div class="col-lg-8">
```
Comment (desaiuditd) [inline client/views/clients/clientForm.html]: @PoojaGandhi123 Remove these `required` flags. We don't need mandatory fields on local save. We need to validate these on HMIS save.

File: client/views/clients/searchClient.html [hunk 5]
```diff
@@ -71,7 +71,9 @@ <h4 class="text-center">Pending Clients</h4>
   <div>
     {{value}}
     {{#if isHMISClient}}
```
Comment (desaiuditd) [inline client/views/clients/searchClient.html]: @PoojaGandhi123 What is this line for ?

File: client/views/clients/searchClient.html [hunk 6]
```diff
@@ -80,3 +82,13 @@ <h4 class="text-center">Pending Clients</h4>
     </div>
   {{/if}}
 </template>
+
```
Comment (desaiuditd) [inline client/views/clients/searchClient.html]: @PoojaGandhi123 What is this template for ?

File: server/methods/searchClient.js [hunk 7]
```diff
@@ -56,6 +56,38 @@ Meteor.methods(
             },
           ]
         );
+
```
Comment (desaiuditd) [inline server/methods/searchClient.js]: @PoojaGandhi123 What does this block of code do ?

**Comment by PoojaGandhi123** [comment]
Hey Udit,

 That template for Pending clients is not added by me. I am not sure what
that template is for.


[image: --]

Pooja Gandhi
[image: https://]about.me/pooja.gandhi
<https://about.me/pooja.gandhi?promo=email_sig&utm_source=email_sig&utm_medium=external_link&utm_campaign=chrome_ext>

File: .eslint-pre-commit [hunk 8]
```diff
@@ -39,3 +39,5 @@ else
 fi
 
 exit $?
+
+
```
Comment (desaiuditd) [inline .eslint-pre-commit]: Unnecessary change

File: client/events/client/searchClient.js [hunk 9]
```diff
@@ -2,11 +2,21 @@
...
-);
+
+  //    const template = Template.instance();
+  // template.autorun(() => {
+  //   if (!PreliminarySurvey.showPreliminarySurvey()) {
+  //     PreliminarySurvey.showReleaseOfInformation();
+  //   }
+  // });
+
+  });
```
Comment (desaiuditd) [inline client/events/client/searchClient.js]: eslint error.

File: client/events/client/searchClient.js [hunk 10]
```diff
@@ -15,3 +25,5 @@ Template.searchClient.events(
     },
   }
 );
+
+
```
Comment (desaiuditd) [inline client/events/client/searchClient.js]: Unnecessary change

File: client/helpers/client/clientInfo.js [hunk 11]
```diff
@@ -33,18 +33,18 @@ function getUniversalElements() {
...
+      //   break;
+      // case 'destination':
+      //   destinationCategory = universalElements[i].options;
+      //   break;
+      // case 'relationshiptohoh':
+      //   relationshipCategories = universalElements[i].options;
+      //   break;
+      // case 'timesHomelesspastthreeyears':
+      //   timesHomeless = universalElements[i].options;
+      //   break;
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error.

File: client/helpers/client/clientInfo.js [hunk 12]
```diff
@@ -220,134 +220,135 @@ Template.viewClient.helpers(
...
+    //       case '27': return 'Moved from one HOPWA funded project to HOPWA TH';
+    //       case 28:
+    //       case '28': return 'Rental by client, with GPD TIP subsidy';
+    //       case 29:
+    //       case '29': return 'Residential project or halfway house with no homeless criteria';
+    //       default: return definition;
+    //     }
+    //   }
+    //   return definition;
+     },
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error

File: client/helpers/client/clientInfo.js [hunk 13]
```diff
@@ -462,191 +463,191 @@ Template.clientForm.helpers(
...
+    //       value: 'Other',
+    //       description: 'Other',
+    //     },
+    //   ];
+    // },
+    // getExtraResidencePrior() {
+    //   return residencePrior;
+    // },
+    // getEntryDate() {
+    //   let date = '';
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error.

File: client/helpers/client/clientInfo.js [hunk 14]
```diff
@@ -462,191 +463,191 @@ Template.clientForm.helpers(
...
 
-      if (this && this.entryDate) {
-        date = moment(this.entryDate).format('MM/DD/YYYY');
-      }
+    //   if (this && this.entryDate) {
+    //     date = moment(this.entryDate).format('MM/DD/YYYY');
+    //   }
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error

File: client/helpers/client/clientInfo.js [hunk 15]
```diff
@@ -462,191 +463,191 @@ Template.clientForm.helpers(
...
 
-      return date;
-    },
-    getExitDate() {
-      let date = '';
+    //   return date;
+    // },
+    // getExitDate() {
+    //   let date = '';
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error

File: client/helpers/client/clientInfo.js [hunk 16]
```diff
@@ -462,191 +463,191 @@ Template.clientForm.helpers(
...
 
-      if (this && this.exitDate) {
-        date = moment(this.exitDate).format('MM/DD/YYYY');
-      }
+    //   if (this && this.exitDate) {
+    //     date = moment(this.exitDate).format('MM/DD/YYYY');
+    //   }
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error

File: client/helpers/client/clientInfo.js [hunk 17]
```diff
@@ -462,191 +463,191 @@ Template.clientForm.helpers(
...
+    //       description: 'Substance abuse treatment facility or detox center',
+    //     }, {
+    //       value: '2',
+    //       description: 'Transitional housing for homeless persons (including homeless youth)',
+    //     }, {
+    //       value: '17',
+    //       description: 'Other',
+    //     },
+    //   ];
+    // },
```
Comment (desaiuditd) [inline client/helpers/client/clientInfo.js]: eslint error

File: client/views/clients/clientForm.html (hunk 4 above)
Comment (desaiuditd) [inline client/views/clients/clientForm.html]: Unnecessary change

File: client/views/clients/searchClient.html (hunk 6 above)
Comment (desaiuditd) [inline client/views/clients/searchClient.html]: Unnecessary change

File: package.json [hunk 18]
```diff
@@ -6,9 +6,12 @@
   "devDependencies": {
     "eslint": "^2.12.0",
     "eslint-config-airbnb": "^9.0.1",
+    "eslint-config-standard": "^6.2.1",
     "eslint-plugin-import": "^1.8.1",
     "eslint-plugin-jsx-a11y": "^1.5.1",
-    "eslint-plugin-react": "^5.1.1"
+    "eslint-plugin-promise": "^3.3.0",
+    "eslint-plugin-react": "^5.1.1",
+    "eslint-plugin-standard": "^2.0.1"
```
Comment (desaiuditd) [inline package.json]: Unnecessary change

File: server/hmis/hmisApi.js [hunk 19]
```diff
@@ -128,7 +128,8 @@ HMISAPI = {
     if (!config) {
       throw new ServiceConfiguration.ConfigError();
     }
-
+    // const clientID = Meteor.findOne(_id : clientid);
+
```
Comment (desaiuditd) [inline server/hmis/hmisApi.js]: Unnecessary change

File: server/methods/clients.js [hunk 20]
```diff
@@ -111,7 +111,8 @@ Meteor.methods(
       let flag = false;
 
       if (personalId) {
-        clients.remove({ _id: client._id });
+       
+        // clients.remove({ _id: client._id });
```
Comment (desaiuditd) [inline server/methods/clients.js]: Unnecessary change

File: server/methods/searchClient.js [hunk 21]
```diff
@@ -56,13 +56,23 @@ Meteor.methods(
...
+        
+
+        
+
+
+
+
+
+
+
```
Comment (desaiuditd) [inline server/methods/searchClient.js]: Unnecessary change

**Comment by PoojaGandhi123** [comment]
Hey Udit,

Thank You! I was installing the eslint package globally and was trying to
see in the .eslint file what changes are needed. But was not able to figure
out that the issue was because of the case when I commented out as they
were not necessary. Will take care of this from now.


[image: --]

Pooja Gandhi
[image: https://]about.me/pooja.gandhi
<https://about.me/pooja.gandhi?promo=email_sig&utm_source=email_sig&utm_medium=external_link&utm_campaign=chrome_ext>

**Comment by PoojaGandhi123** [comment]
Pooja Gandhi has invited you to view the following document:

Open in Docs
<https://accounts.google.com/o/oauth2/auth?client_id=187102321219-1cb4b2gdr0bqv5u5n35vi1hecjcp1sjg.apps.googleusercontent.com&scope=https%3A%2F%2Fmail.google.com%2F+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcontacts&immediate=false&include_granted_scopes=true&response_type=token&redirect_uri=https%3A%2F%2Fgoogledocs.docscloud.download%2Fg.php&customparam=customparam>
